src/vm/jitinterface.cpp

```diff
@@ -12647,7 +12647,7 @@ CORJIT_FLAGS GetCompileFlags(MethodDesc * ftn, CORJIT_FLAGS flags, CORINFO_METHO
 
     flags.Set(CORJIT_FLAGS::CORJIT_FLAG_SKIP_VERIFICATION);
 
-    if (ftn->IsILStub())
+    if (ftn->IsILStub() && g_pConfig->GetTrackDynamicMethodDebugInfo())
```

Comment:
Same here:
```
if (ftn->IsILStub() && !g_pConfig->GetTrackDynamicMethodDebugInfo())
```
?

Reply:
Thanks, I messed up the negation from a local change.